src/libtomahawk/audio/AudioOutput.cpp

```diff
@@ -37,7 +37,7 @@
 #include <vlc/libvlc_events.h>
 #include <vlc/libvlc_version.h>
 
-static const int ABOUT_TO_FINISH_TIME = 2000;
+static const float ABOUT_TO_FINISH_POS = 0.95f;
```

Comment:
I'm happy to get rid of it, if we can. It used to be important, but that stems from old phonon-vlc times.